Module 5/Ex0ToEx5 V2/Includes/OfficeBlock.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef OFFICEBLOCK_HPP
+#define OFFICEBLOCK_HPP
+
+#include "Bureaucrat.hpp"
+#include "Intern.hpp"
+#include <memory>
+
+class OfficeBlock {
+public:
+  OfficeBlock();
+  OfficeBlock(std::unique_ptr<Intern> intern, std::unique_ptr<Bureaucrat> signing, std::unique_ptr<Bureaucrat> executing);

```

Comment:
The copy constructor is implicitly defined if not deleted specifically.
The class having pointers as members, this will create issues if the default copy constructor is used.